packages/amplify-graphql-transformer-core/src/transform-host.ts

```diff
@@ -187,6 +230,28 @@ export class DefaultTransformHost implements TransformHostProvider {
     pipelineConfig?: string[],
     scope?: Construct,
     runtime?: CfnFunctionConfiguration.AppSyncRuntimeProperty,
```

Comment:
same here: should we let this method handle the runtime setting internally?

Reply:
Addressed in 27f4ea151210acf0bdde352e52068be27b83ecc7